docs/index.rst

```diff
@@ -6,8 +6,220 @@
 The Zoo
 =======
 
+A smart service catalog providing an overview of your services' development and operations.
+
+Adding custom checks
+====================
+
+If you want to add your own checks to the zoo, you need to create your own project with files:
```

Comment:
Add information about the structure. It should be a python package with two directories inside, `metadata` and `checks`. Inside the `metadata` folder will go the `.yml` files and inside the `checks` folder will go the python functions that actually perform the checks.